batch/batch_test.go

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+package batch
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/helpers"
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/resources"
+)
+
+var (
+	resourceName string
+	clientID     string
+	clientSecret string
+	accountName  string
+	jobID        string
+	poolID       string
+)
+
+func TestMain(m *testing.M) {
+	err := parseArgs()
+	if err != nil {
+		log.Fatalln("failed to parse args")
+	}
+	os.Exit(m.Run())
+}
+
+func parseArgs() error {
+	err := helpers.ParseArgs()
+	if err != nil {
+		return fmt.Errorf("cannot parse args: %v", err)
+	}
+
+	accountName = os.Getenv("AZ_BATCH_NAME")
+	if !(len(resourceName) > 0) {
+		accountName = strings.ToLower("b" + helpers.GetRandomLetterSequence(10))
+	}
+
+	jobID = strings.ToLower("j" + helpers.GetRandomLetterSequence(10))
+	poolID = strings.ToLower("p" + helpers.GetRandomLetterSequence(10))
+
+	clientID = os.Getenv("AZ_CLIENT_ID")
+	clientSecret = os.Getenv("AZ_CLIENT_SECRET")
+
+	return nil
```

Comment:
What are this used for?

Reply:
Looks like they sneaked in when copied the example from AKS sample. Will remove